docs/guides/docker.md

```diff
@@ -162,9 +162,9 @@ We can check that our build worked by running a container from our new image.
 
 ## Deploying to a remote environment
```

Comment:
Somewhere after this header, the rendering gets very wonky

Reply:
Thank you! Showing better now in preview. Adjusted spacing. I checked a number of code snippets on other pages afterward and nothing jumped out as off.

Changes pushed.

Reply:
One sec.

Reply:
Fixed:

<img width="679" alt="Screenshot 2023-09-08 at 4 55 43 PM" src="https://github.com/PrefectHQ/prefect/assets/7703961/05394544-140c-4d8c-9f35-30ee53ee0735">